model/database.php

```diff
@@ -19,12 +19,11 @@ class Database
      */
     function __construct()
     {
-        try{
+        try {
             //Creates a new PDO connection
-            $this ->_dbh = new PDO(DB_DSN, DB_USERNAME, DB_PASSWORD);
-        }
-        catch (PDOException $e){
-            echo $e -> getMessage();
+            $this->_dbh = new PDO(DB_DSN, DB_USERNAME, DB_PASSWORD);
+        } catch (PDOException $e) {
```

Comment:
we should comment this out since we know the connection works now. Unless you would like to display a page stating sorry but we ran into a error

Reply:
We should write a standard view for errors at some point and redirect all these types of failures there, but I think it is out of scope for this branch.

I cant get rid of the catch though, it has to be there.